linkerd/datadog_checks/linkerd/linkerd.py

```diff
@@ -21,3 +21,7 @@ def __init__(self, name, init_config, agentConfig, instances=None):
             'linkerd': {'labels_mapper': labels_mapper, 'metrics': [METRIC_MAP], 'type_overrides': TYPE_OVERRIDES}
         }
         super(LinkerdCheck, self).__init__(name, init_config, agentConfig, instances, default_config, 'linkerd')
+
+    def service_check(self, name, status, tags=None, hostname=None, message=None):
```

Comment:
Do we need to define health_service_check config somewhere?

Reply:
No it's already done by `OpenMetricsBaseCheck`.